subcommittees/mentoring/files/discussion_recruiting_email.txt

```diff
@@ -0,0 +1,13 @@
+Greetings, all!  
+
+This is an invitation to contribute to the Software and Data Carpentry communities in a very low-commitment, lots-of-fun way!  
```

Comment:
> very low-commitment, lots-of-fun way!

The lots-of-fun way is **TRUE**!!! I'm not sure about the very low-commitment. Host need to show up on their sessions. But I'm confident that any of our amazing instructors will let us down.